File: src/main/java/ru/zdoher/japs/repositories/PartOfSpeechRepository.java
package ru.zdoher.japs.repositories;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import ru.zdoher.japs.domain.PartOfSpeech;

@Repository
public interface PartOfSpeechRepository
        extends CrudRepository<PartOfSpeech, String> {
}

File: src/main/java/ru/zdoher/japs/repositories/PartOfSpeechRepository.kt
package ru.zdoher.japs.repositories

import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository
import ru.zdoher.japs.domain.PartOfSpeech

@Repository
interface PartOfSpeechRepository : ReactiveMongoRepository<PartOfSpeech, String>